### PR TITLE
Add protips from TT-kamerat general chat

### PR DESCRIPTION
**Pull request by atuukkanen**

### CODE REVIEW

File: src/masterlist.py [hunk 1]
```diff
@@ -137,7 +137,49 @@
...
     "Oikeisiin kuvauksiin sit Canonit.",
-    "Ninja walk roskiin"
+    "Ninja walk roskiin",
+    "Kun kuvaat lumista maisemaa niin muista kompensoida valotusta vähintään yhdellä stopilla.",
+    "Jos lähdet kuvaamaan ja filmissä on jäljellä vähemmän kuin viisi ruutua, vaihda filmi.",
+    "Aina mukaan vähintää kolme kameraa eri setupeil et saa lauottuu joka sormel eritahtiin ja yks jäähtyy ettei suljin polta filmii.",
+    "Take your time become a better photographer and learn more about editing software",
+    "Voit ajatella ton ostamisen sijoituksena inspiraatioon",
+    "There are no presets \"one size fits all\" I had downloaded 500 of these 99% are worthless.",
+    "älä osta halpoja sdi kaapeleita, edes batch kaapeleiksi",
```
Comment (kulmajaba) [inline src/masterlist.py:147]: Fix typo: batch->patch

File: src/masterlist.py [hunk 2]
```diff
@@ -137,7 +137,49 @@
     "Uljaat prinssit kuvataan alaviistosta.",
     "Jos löydät houkuttelevan tarjouksen, sovi kaupat välittömästi ja mieti logistiikkaa vasta sen jälkeen.",
     "Oikeisiin kuvauksiin sit Canonit.",
-    "Ninja walk roskiin"
+    "Ninja walk roskiin",
+    "Kun kuvaat lumista maisemaa niin muista kompensoida valotusta vähintään yhdellä stopilla.",
+    "Jos lähdet kuvaamaan ja filmissä on jäljellä vähemmän kuin viisi ruutua, vaihda filmi.",
+    "Aina mukaan vähintää kolme kameraa eri setupeil et saa lauottuu joka sormel eritahtiin ja yks jäähtyy ettei suljin polta filmii.",
+    "Take your time become a better photographer and learn more about editing software",
```
Comment (nippis) [inline src/masterlist.py:144]: Pilkku pls: "take your time, become a better"

Comment (atuukkanen) [reply src/masterlist.py:144]: Suora lainaus, ei pysty

File: src/masterlist.py (hunk 1 above)
Comment (atuukkanen) [reply src/masterlist.py]: Direct quote but fixing this typo did not change the tone too much so did it.

**Comment by eltsu7** [comment]
huoh

**Pull request merged**